packages/deprecation-crawler/src/lib/commands/default.command.ts

```diff
@@ -24,19 +24,22 @@ export const defaultCommand: YargsCommandObject = {
             (config: CrawlConfig): CrawlerProcess =>
               askToSkip('Grouping?', group(config), {
                 precondition: async (r) =>
-                  r.deprecations?.some((d) => !d.group),
+                  r.deprecations?.some((d) => !d.group)
               }),
-            generateOutput,
-            updateRepository,
+            (config: CrawlConfig): CrawlerProcess =>
```

Comment:
Imho it shouldn't be a choice, I think we should always format here?

Reply:
Agree!